packages/js/plugins/uri-resolvers/ens-resolver/src/__tests__/helpers/getClient.ts

```diff
@@ -17,82 +17,81 @@ import {
   defaultInterfaces,
   defaultPackages,
   defaultIpfsProviders,
-  defaultWrappers
+  defaultWrappers,
 } from "@polywrap/client-config-builder-js";
 import { httpPlugin } from "@polywrap/http-plugin-js";
 import { Uri } from "@polywrap/core-js";
 
 export const getClient = () => {
```

Comment:
i guess these `getClient` functions can be improved to use the builder with default? And just adding what we need

Reply:
For sure. My main focus here was just to "get this working". I'd say any other refactors such as these will happen as we see the need for them arise. I know some like to set up their config manually, without builder